apps-rendering/src/client/article.ts

```diff
@@ -92,10 +92,10 @@ function followToggle(
 				);
 			});
 		} else {
-			void bridgetClient.follow(topic).then((_) => {
+			void bridgetClient.follow(topic).then((following) => {
 				ReactDOM.render(
 					h(followStatusComponent, {
-						isFollowing: true,
+						isFollowing: following,
```

Comment:
If you’re keen, you can name your argument `isFollowing` and use the shorthand notation:

```ts
(isFollowing) => /* truncated */ ({ isFollowing })
// is equivalent to
(isFollowing) => /* truncated */ ({ isFollowing: isFollowing })
```